nion/swift/ImageCanvasItem.py

```diff
@@ -1127,7 +1221,13 @@ def mouse_released(self, x: int, y: int, modifiers: UserInterface.KeyboardModifi
         if self.__mouse_handler:
             self.__mouse_handler.mouse_released(Geometry.IntPoint(y, x), modifiers)
             self.__mouse_handler = None
-        if delegate.tool_mode != "hand":
+
+        # Should probably wrap this into a function of 'Non-Toggle' UI elements
```

Comment:
This comment is not wrong, but I'd prefer to just make another PR with the change, or put it in an issue instead of making to-do items in the code that will never get done. I used to do the same thing, and almost none of them get done, so my preference now is to file an issue for "internal maintenance."